src/App.vue

```diff
@@ -109,18 +118,28 @@ export default {
   },
   methods: {
     ...mapActions('elementalChat', ['setChannelPolling', 'updateProfile']),
-    ...mapActions('holochain', ['skipBackoff']),
+    ...mapActions('holochain', ['skipBackoff', 'holoLogout']),
     ...mapMutations(['setAgentHandle', 'setErrorMessage']),
+    ...mapMutations('elementalChat', ['setNeedsHandle']),
     agentHandleEntered () {
       if (this.internalAgentHandle === '') return
       this.updateProfile(this.internalAgentHandle)
       this.dialog = false
     },
+    closeModal () {
+      this.setNeedsHandle(false)
+    },
     clearErrorMessage () {
       this.setErrorMessage('')
     },
     retryNow () {
       this.skipBackoff()
+    },
+    disconnectedHoloLogout () {
+      log('refreshed page to lose display of data')
+      window.history.go(0)
```

Comment:
Might be more idiomatic to do `window.location.reload()` but I've never used it so maybe not worth risking breaking anything

Reply:
I tried this, but it's apparently not compatible with vue. Under the hood in the vue router, uses window.history.go, so I decdied to also use this approach.